pandas/core/indexes/period.py

```diff
@@ -161,6 +161,24 @@ def to_timestamp(self, freq=None, how="start") -> DatetimeIndex:
         arr = self._data.to_timestamp(freq, how)
         return DatetimeIndex._simple_new(arr, name=self.name)
 
+    # error: Decorated property not supported  [misc]
```

Comment:
also need to remove from inherit_names?

and maybe update comment on L151

Reply:
updated comment; removing from inherit_names is a PITA since it isnt there directly.  harmless to have it there.